src/pages/index.astro

```diff
@@ -146,6 +172,58 @@ section.hero {
   flex: 1 1 20rem;
 }
 
+.updates-heading {
+  margin-bottom: var(--spacing-lg);
+}
+
+.updates-carousel {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  gap: var(--spacing-md);
+  margin-bottom: var(--spacing-lg);
+}
+
+.updates-carousel .carousel-arrow {
```

Comment:
We can add button styles by using these classes in the element `btn` and `flexible`
Then we can style the button and the svg separately, like so

```
.updates-carousel .carousel-arrow {
  padding: 0;
  border: none;
  background-color: inherit;
  color: var(--button-carousel-color);
}

.updates-carousel svg {
  height: 1.25rem;
  width: 1.25rem;
  display: inline-block;
}
 ```